server.js

```diff
@@ -17,115 +26,119 @@ const options = {
 }
 // ☞ bd397750-457f-4308-b616-f0424ddc5d04
 
-mongoose.connect('mongodb://ds141611.mlab.com:41611/d2rd-notes', options)
-.then(() => console.log('Success connecting the MongoDB/d2rd-notes on mlab'))
+mongoose.connect(activeDB, options)
+.then(() => console.log('Success connecting the MongoDB/notes on mlab'))
 .catch((err) => console.log(err.message)) // TEST: changing PW should throw 'authentication failed error
-
+// NOTE: EACH DB HAS A UNIQUE CONNECTION STRING
 // create schema
 // ☞ 8cf866c9-a061-48df-a275-ebdbf2196f60
 // REFACTORED TO MOVE NOTES TO MONGODB
 
+//TEST DATA
+const newTestNote = {
+  "title": "Yamaha SLB-200LTD Silent Bass",
+  "priority": 3,
+  "body": "The most popular Electric Upright Bass in the line-up."
+};
+
+
+//USING AXIOS REQUESTS 3-21-19
+Axios.get(mLabNotes)
```

Comment:
You can put your endpoints here, but I would almost put them in another file -- call it noteController.js or noteRouter.js


in it, at the top, you would const router  = require("express").Router();

then create  (req, res) functions that do the CRUD. 

then module.export the router. And then require that file in your server.js. 

Then you would `server.use("/api/notes", noteRouter) `as global middleware in your server.js file.

Reply:
And line 46 `Axios.get(mLabNotes) can just be deleted?  Is this line even functional in server.js, or is it being ignored?